tools/json/viyalab_mizu.json

```diff
@@ -9,7 +9,7 @@
     },
     "core": "earlephilhower",
     "cpu": "cortex-m0plus",
-    "extra_flags": "-D ARDUINO_DEGZ_MIZU -DARDUINO_ARCH_RP2040 -DUSBD_MAX_POWER_MA=250",
+    "extra_flags": "-D ARDUINO_VIYALAB_MIZU_RP2040 -DARDUINO_ARCH_RP2040 -DUSBD_MAX_POWER_MA=250",
```

Comment:
Do you want to keep the other define `-D ARDUINO_DEGU_MIZU` too?  That way existing code won't need changes.

Reply:
No. We changed the brand. I just want to change the name of the board. It's same board.